serialization/__init__.py

```diff
@@ -0,0 +1,2 @@
+from base import *
+from pytorch import *
```

Comment:
This code is wrong. Did you test this? I wonder how this could have worked without an exception.
```suggestion
from .base import *
from .pytorch import *
```
However, I'm not sure if we really want to import everything here?
Maybe only the stuff from `base`?
```suggestion
from .base import *
```

Reply:
Ah sure this is wrong